### PR TITLE
WYS Analyze Null Data

### PR DESCRIPTION
**Pull request by KatiRG**
## What this pull request accomplishes:
- Determines date ranges for each WYS sign for which speed and count columns are NULL
- Tallies the number of signs containing NULL speed and count columns

## Issue(s) this solves:

- #393 

## What, in particular, needs to reviewed:

- approve the new text added to the README file at the end in a new section called `Quality Checks` (the notebook has already been reviewed and approved in the issue)

### CODE REVIEW

File: wys/api/README.md [hunk 1]
```diff
@@ -100,3 +100,14 @@ The data is inserted into `wys.raw_data`. Data from the API is already pre-aggre
...
 `dow`|integer|Day of the week. Sunday is 0, and Saturday is 6|4
+
+## Quality Checks
+An analysis on 23.4.2021 to investigate rows with NULL speed and count columns was performed in notebook `investigate_api_nulls.ipynb` in this directory, as part of [issue #393](https://github.com/CityofToronto/bdit_data-sources/issues/393). Main findings:
+
+- there are currently `872` rows in `wys.raw_data`  
+- `727` of these contain rows with NULL speed or count columns  
+- `739` signs have been operating since `> 2021-03-31`
+- of the `734` signs have operating since `> 2021-03-31`, `734` have NULL speed or count columns
+- only `395` rows have matching geoms in `wys.wys_stat_signs_speed_limit`
```
Comment (radumas) [inline wys/api/README.md:111]: this is irrelevant to this. If you want to use a view of stationary signs, please use `wys.stationary_signs`.

**Review by radumas — CHANGES_REQUESTED** [review]
Thanks. A few clarifying requests.

File: wys/api/README.md (hunk 1 above)
Comment (KatiRG) [reply wys/api/README.md:111]: removed

**Review by radumas — APPROVED** [review]
🚀

**Pull request merged**